app/controllers/checkin_controller.rb

```diff
@@ -1,12 +1,16 @@
 class CheckinController < ApplicationController
   before_action :require_login
 
+  def new
+    flash.clear
+  end
+
   def create
     flash.clear
     Time.zone = 'Pacific Time (US & Canada)'
     @user = Student.find_by_id(session[:current_user_id])
     @checkin = Checkin.new
-    @checkin.update(time: DateTime.now, student_id: @user.id)
+    @checkin.update(time: DateTime.now, student_id: @user.id, reason: params[:checkin][:reason])
```

Comment:
I'm using `params[:checkin][:reason]` to get the reason from the form as `form_with ` (using a model) generates a request header like so:
<img width="1155" alt="Screen Shot 2022-03-28 at 10 01 29 AM" src="https://user-images.githubusercontent.com/69605001/160450603-e70dd8fe-11b4-4928-909e-eab2d296eb8f.png">